server/tserver/src/main/java/org/apache/accumulo/tserver/ThriftClientHandler.java

```diff
@@ -1454,9 +1454,16 @@ public void run() {
     }
   }
 
+  @Deprecated
   @Override
   public void unloadTablet(TInfo tinfo, TCredentials credentials, String lock, TKeyExtent textent,
       TUnloadTabletGoal goal, long requestTime) {
+
```

Comment:
For RPC compatibility for rolling restarts, you should probably have a real implementation here, by calling the new unload method with the toString version of the goal.